scripts/handle-user-data

```diff
@@ -35,11 +35,12 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
-    # Here we used bind mount because currently openshift doesn't work
```

Comment:
> this comment is now obsolete? So we have a single partition now? No need for mount bind? 

Right this is now obsolete.

> What's about the Docker config and images?

docker config and images are already persistent across restart because `/var/lib/docker` already have soft link to persistent storage side https://github.com/minishift/minishift-centos-iso/blob/master/scripts/handle-user-data#L30